packages/blockchain-api/src/contractKitDataSource.ts

```diff
@@ -0,0 +1,11 @@
+import { ContractKit, newKit } from '@celo/contractkit'
+import CacheDataSource from './cacheDataSource'
+
+export default class ContractKitDataSouce extends CacheDataSource<any> {
```

Comment:
Is it potentially problematic if CK results are cached? Like a balance changes on-chain but the CK data source returns a cached value?

Reply:
It's up to the consumer to define cache keys that appropriately consider expiration. I.e. including block or epoch numbers in the cache key should be part of it